loadData/rental_data.rb

```diff
@@ -0,0 +1,23 @@
+require 'json'
+require_relative '../school/book'
+require_relative '../school/person'
+require_relative '../school/rental'
+
+@file_rental = File.read('./data/rental.json')
+
+def rental_data(rental)
+  return if @file_rental.empty?
+
+  data_rental = JSON.parse(@file_rental)
+  data_rental.map do |value|
+    age = value['Age']
+    name = value['Name']
+    book = value['Book']
+    author = value['Author']
+    date = value['Date']
+    book_info = Book.new(book, author)
+    person_info = Person.new(age, name)
+    rental_info = Rental.new(date, book_info, person_info)
+    rental << rental_info
+  end
+end
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/176337871-2f552343-f936-4387-8fcc-fe260e226529.png)

- [ ] Great job on saving data in JSON files, However, in the requirements it said that you should handle the case where one of the JSON files is missing the application should not throw an error. I deleted the `/data/rental.json` file and the app crashed. Check this hint solution :point_down: 

```rb
# Kindly note that this is just an example solution, feel free to solve it the way you want, just make sure the app does not  crash when this file is missing

def rental_data(rental)
  File.write('./data/rental.json', JSON.pretty_generate([])) unless File.exists?('./data/rental.json')
  @file_rental = File.read('./data/rental.json') 
  return if @file_rental.empty?

  data_rental = JSON.parse(@file_rental)
  data_rental.map do |value|
    age = value['Age']
    name = value['Name']
    book = value['Book']
    author = value['Author']
    date = value['Date']
    book_info = Book.new(book, author)
    person_info = Person.new(age, name)
    rental_info = Rental.new(date, book_info, person_info)
    rental << rental_info
  end
end

```